CDT/include/Triangulation.hpp

```diff
@@ -426,7 +426,9 @@ namespace detail
 template <typename T>
 T lerp(const T& a, const T& b, const T t)
 {
-    return (T(1) - t) * a + t * b;
+    if((a <= 0 && b >= 0) || (a >= 0 && b <= 0))
```

Comment:
Why is this change necessary?

Reply:
not necessary. I was trying to reduce rounding errors, but that wasn't the actual problem.